packages/firebase_core_tizen/example/tizen/src/runner.cc

```diff
@@ -0,0 +1,18 @@
+#include "runner.h"
```

Comment:
Please consider .Net host first in the Example.
```
# Create a new C++ app project in "app_name" directory.
# Typically C++ apps consume less memory than C# (default) apps, but are not compatible with TV devices.
flutter-tizen create --tizen-language cpp app_name
```

Reply:
FYI) The official term we use in our documentation is "app language", which is either C++ or C#.

Reply:
The runner does not matter. Switched to C# app language.